app/services/creators/special_group_creator.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+#
+# Service object for 'special' (non-draw) housing groups
+class SpecialGroupCreator < Creator
+  # Initialize a new SpecialGroupCreator
+  #
+  # @param params [#to_h] the params for the group
+  def initialize(params)
+    super(klass: Group, name_method: :name, params: params)
+    process_params
+  end
+
+  private
+
+  def process_params
+    @params = params.to_h.transform_keys(&:to_sym)
+    remove_blank_members if params[:member_ids]
+    remove_members_and_leader_from_draw
+  end
+
+  def remove_blank_members
+    @params[:member_ids] = params[:member_ids].reject(&:empty?)
+  end
```

Comment:
Hmm, the alternative could be just to use GroupCreator and add some `draw.present?` logic? I'll ping you on Slack to discuss.